Add the “Focus Tests” exercise solution

chasenlehara opened the pull request:
This adds just the solution for the “Focus Tests” exercise, a part of the “Incorporating Accessibility Into Your Development Process” training.

The main app code is copied from [another exercise](https://github.com/bitovi/trainings/tree/063dc037131104342f9b320f2da6dfdb99d666ae/typing-react-state/Exercise3/solution). The only addition is the test file that verifies a part of the app.

I’ll add the `problem` folder in a follow-up commit (once this gets reviewed).

Solution on CodeSandbox: https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/focus-tests/incorporating-accessibility-into-your-dev-process/focus-tests/solution?file=src/App.test.tsx

arcoleburn commented:
so the task here is to finish writing a partially written test to check if an element has focus?

chasenlehara commented:
> so the task here is to finish writing a partially written test to check if an element has focus?

Yeah, I think I would leave the comments in the test and the exercise would be to write the assertions.

I’m reconsidering this since I wrote it three weeks ago—it’s boring to write tests. I would still like to get this code reviewed and merged in, and later I will figure out if I’ll include this exercise as-is, or just use it to do some live coding, or scrap it.

chasenlehara committed:
Add the “Focus Tests” exercise solution

This adds just the solution for the “Focus Tests” exercise, a part of the “Incorporating Accessibility Into Your Development Process” training.

The main app code is copied from [another exercise](https://github.com/bitovi/trainings/tree/063dc037131104342f9b320f2da6dfdb99d666ae/typing-react-state/Exercise3/solution). The only addition is the test file that verifies a part of the app.

I’ll add the `problem` folder in a follow-up commit (once this gets reviewed).
